### PR TITLE
MetaOS readme changes and issue fixes

### PR DESCRIPTION
**Pull request by Pawan-MSFT**

### CODE REVIEW

File: samples/app-complete-sample/csharp/manifest_hub/manifest_hub.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://{{domain-name}}/Page/index.html",
+      "canUpdateConfiguration": false,
+      "scopes": [ "team", "groupchat" ]
+    }
+  ],
+  "bots": [
+    {
+      "botId": "{{Microsoft-App-Id}}",
```
Comment (ChetanSharma-msft) [inline samples/app-complete-sample/csharp/manifest_hub/manifest_hub.json]: Remove these sections.

File: samples/app-complete-sample/nodejs/Manifest_hub/manifest.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  "description": {
+    "short": "Bot Template - a template for creating a bot",
+    "full": "The Bot Template is meant to create an environment in which to easily create a bot."
+  },
+  "icons": {
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "accentColor": "#0079bf",
+  "configurableTabs": [
```
Comment (ChetanSharma-msft) [inline samples/app-complete-sample/nodejs/Manifest_hub/manifest.json:26]: Remove configurable tab and bots

File: samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.13",
+  "id": "29f3f5e0-2674-4685-b919-4ea5bbed6ed2",
+  "version": "1.0.0",
+  "packageName": "com.custom.tab",
+  "developer": {
+    "name": "Contoso",
+    "websiteUrl": "https://6dd4-14-142-36-202.ngrok.io",
```
Comment (ChetanSharma-msft) [inline samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json:9]: Please update these URls

File: samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "accentColor": "#D85028",
+  "configurableTabs": [],
+  "staticTabs": [
+    {
+      "entityId": "index",
+      "name": "Personal Tab",
+      "contentUrl": "https://6dd4-14-142-36-202.ngrok.io/PersonalTab",
```
Comment (ChetanSharma-msft) [inline samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json:32]: Please update these URls

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Comment by ChetanSharma-msft** [comment]
Please run build pipeline.

File: samples/app-complete-sample/csharp/manifest_hub/manifest_hub.json (hunk 1 above)
Comment (Pawan-MSFT) [reply samples/app-complete-sample/csharp/manifest_hub/manifest_hub.json]: Fixed

File: samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json (hunk 3 above)
Comment (Pawan-MSFT) [reply samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json]: Fixed

File: samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json (hunk 4 above)
Comment (Pawan-MSFT) [reply samples/tab-personal/mvc-csharp/Manifest_Hub/manifest.json]: Fixed

**Comment by Pawan-MSFT** [comment]
**Execute build pipeline:** 
[https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/4005963631
Result:](https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/4005963631)

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

**Comment by ChetanSharma-msft** [comment]
Execute build pipeline:

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**